.gitignore

```diff
@@ -1,3 +1,5 @@
+This should be ignored by git
+test.md
 # Logs
 logs
 *.log
```

Comment:
- [ ] As the last code reviewer said: 

> Great job 👍 adding the node modules to the gitignore file, but I see that you didn't put the test.md file inside the gitignore, so kindly add the test.md file inside the gitignore as it also part of the requirement.

- [ ] Please 🙏🏽 add `./test.md` to the `.gitigore` file, to ignore the file by git,  when you correctly added the file to the `.gitignore` the test.md file doesn't charge in your repository:

![image](https://user-images.githubusercontent.com/58642949/212050281-583c35a1-8416-4dd9-9434-4bc123ca8cc2.png)